src/Deprecated12/SHRBTextStyler.extension.st

```diff
@@ -19,12 +19,3 @@ SHRBTextStyler class >> initialTextAttributesForPixelHeight: aNumber [
 	self deprecated: 'Use #initialTextAttributes instead.' transformWith: '`@rcv initialTextAttributesForPixelHeight: `@arg1' -> '`@rcv initialTextAttributes'.
 	^ self initialTextAttributes
 ]
-
-{ #category : #'*Deprecated12' }
-SHRBTextStyler class >> initialTextAttributesForPixelHeight: aNumber styleTable: aStyleTable [
-
-	self
-		deprecated: 'Use #initialTextAttributesForStyleTable: instead.'
-		transformWith: '`@rcv initialTextAttributesForPixelHeight: `@arg1 styleTable: `arg2' -> '`@rcv initialTextAttributesForStyleTable: `@arg2'.
-	^ self initialTextAttributesForStyleTable: aStyleTable
-]
```

Comment:
Is this removal intentional?

Reply:
no, and I guess I have to close this PR again as I have no clue how to fix it, as the image I used I did a spec pr before and spec thinks that it owns this method

Reply:
So how do I fix this PR? I do not want to redo that trivial copy and past yet again...

Reply:
In my image, this method is in Deprecated12, I can even edit it, but when I commit it is ingored

Reply:
https://stackoverflow.com/questions/215718/how-do-i-reset-or-revert-a-file-to-a-specific-revision

This seems to be what you need

Reply:
that's too complex for me (and will nor work, as there is no verson in git as iceberg just does not see this change)